README.md

```diff
@@ -201,14 +212,190 @@ output = model.generate(
 You can also deploy the server to a cloud provider like Jina Cloud or AWS.
 To do so, you can use `deploy` command:
 
-- Jina Cloud
+### Jina Cloud
+
+using predefined executor
 
 ```bash
-opengpt deploy stabilityai/stablelm-tuned-alpha-3b --device cuda --precision fp16 --provider jina --name opengpt --replicas 2
+opengpt deploy yahma/llama-7b-hf --adapter jinaai/alpaca-lora --precision fp16 --cloud jina --replicas 2
 ```
 
-**TBD ...**
+or using customized YAML file
+
+```bash
+opengpt deploy --cloud jina --config flow.yml
```

Comment:
where is the `flow.yml`?

Reply:
One principle, keep the content as simple as possible in our readme. All of the advanced and detailed content will be included in our documentation (available soon).